fastlane/Deliverfile

```diff
@@ -17,8 +17,18 @@ privacy_url({
   'default' => 'https://simplenote.com/privacy/',
 })
 
-copyright('2020 Automattic')
+copyright('2021 Automattic')
```

Comment:
FWIW we also do it manually in other apps, so it's ok to keep it hardcoded, after all we only need to updated it once a year, so I am not sure it's worth it trying to auto-compute it as you pondered about in the PR description.

I'm surprised though that `fastlane deliver` didn't warn you about it sooner about it being outdated 🤔

Reply:
Yeah... 🤔 It's not unlikely it did warn before and I didn't notice it in the noise of the output 😳 .